src/services/llms/openai.py

```diff
@@ -19,11 +19,13 @@ def create_prompt(self, file: PatchedFile, hunk: Hunk, pr_details: PRDetails) ->
         """Create a prompt formatted for OpenAI's expectations."""
         return f"""Your task is reviewing pull requests. Instructions:
         - Provide the response in following JSON format: {{"reviews": [{{"lineNumber": <line_number>, "reviewComment": "<review comment>"}}]}}
+
         - Provide comments and suggestions ONLY if there is something to improve, otherwise "reviews" should be an empty array.
         - Use GitHub Markdown in comments
         - Focus on bugs, security issues, and performance problems
         - IMPORTANT: NEVER suggest adding comments to the code
-        - Comment with language of human: {Config.HUMAN_LANGUAGE}
+
+        Please provide your review in {Config.HUMAN_LANGUAGE} language.
```

Comment:
Việc thay đổi này từ `Comment with language of human: {Config.HUMAN_LANGUAGE}` sang `Please provide your review in {Config.HUMAN_LANGUAGE} language.` có vẻ không cần thiết và có thể gây ra vấn đề về nhất quán.  Nếu mục đích là làm cho hướng dẫn rõ ràng hơn, hãy cân nhắc sửa đổi lại câu văn sao cho ngắn gọn và dễ hiểu hơn, mà vẫn giữ được ý nghĩa ban đầu. Ví dụ: `Hãy đánh giá bằng tiếng {Config.HUMAN_LANGUAGE}`.  Hoặc nếu `Config.HUMAN_LANGUAGE` không phải lúc nào cũng là tiếng Việt, cần xem xét cách diễn đạt tổng quát hơn.